### ISSUE TITLE
Expose isElementAtBottom as a property?

### ISSUE THREAD

**Issue by justin-calleja** [ankeetmaini/react-infinite-scroll-component#41]
Hi,

I am developing with the dev tools open and "toggle device toolbar" on (to display the viewport as a mobile device. (Also, I'm using `scrollableTarget`).

With this setup, `next` is never firing. i.e. the following is always returning `false`:

```js
  isElementAtBottom (target, scrollThreshold = 0.8) {
    const clientHeight = (target === document.body || target === document.documentElement)
    ? window.screen.availHeight : target.clientHeight;

    const scrolled = scrollThreshold * (target.scrollHeight - target.scrollTop);
    return scrolled * (window.devicePixelRatio || 1) <= clientHeight;
  }
```

Can you please confirm that this is an issue (and that I'm not using the component improperly)?

If you'd like the change, I can look into exposing `isElementAtBottom` as a property (shouldn't be hard to do but I haven't really looked at all the code).

Thank you

**Comment by justin-calleja** [comment]
Also, I would like this as a property because if I overwrite it with:

```js
const element = scrollableTarget;
const scrollThreshold = 0.8

return (element.scrollHeight * scrollThreshold) - element.scrollTop <= 0
```

it's returning true when I want it to.

**Comment by ankeetmaini** [comment]
Thanks, but I still feel the component should keep the logic of whether it's there at the bottom or not. If it's not firing next then there's a bug with this component. Can you share a codesandbox or something where I can debug this stuff?

**Comment by ankeetmaini** [comment]
@kdepp thanks for looking into this. I just reverted the device pixel change and published a patch version `v3.0.2`. I've tried out examples at https://codesandbox.io/s/k90wzm7pnv, they're working.

@justin-calleja thanks for raising this up, please re-open this if it still doesn't work.

**Comment by justin-calleja** [comment]
@ankeetmaini sorry for not responding. I'll try it out soon and confirm. Thanks.
